File: Java/problem1455.java
class Solution {
    public int isPrefixOfWord(String sentence, String searchWord) {
        String[] strs = sentence.split(" ");
        int i = 0;
        for (; i < strs.length; ++i) {
            if (strs[i].startsWith(searchWord)) {
                break;
            }
        }
        if (i < strs.length) {
            return i + 1;
        }
        return -1;
    }
}

File: python/problem1455.py
class Solution:
    def isPrefixOfWord(self, sentence: str, searchWord: str) -> int:
        strs = sentence.split()
        i, n = 0, len(strs)
        while i < n:
            if strs[i].startswith(searchWord):
                break
            i += 1
        return i + 1 if i < n else -1
